process_win.cpp

```diff
@@ -192,11 +192,11 @@ void Process::close_fds() {
   
   if(stdin_fd)
     close_stdin();
-  if(stdout_fd) {
+  if(stdout_fd && *stdout_fd != NULL) {
```

Comment:
Thank you. I think the `unique_ptr::reset`'s should be called even when the file descriptors are NULL though. Maybe here for instance write:
```c++
if(stdout_fd) {
  if(*stdout_fd!=NULL) CloseHandle(*stdout_fd);
  stdout_fd.reset();
}
```